### ISSUE TITLE
Tournament

### ISSUE THREAD

**Issue by Bearinx** [WarzoneMC/Website#6]
Add a section on the website to show the tournament date and time, and also to display the brackets.

**Comment by jellz** [comment]
Yes! [Here's a good tutorial on how to do it](https://feedback.challonge.com/knowledgebase/articles/257732-embedding-brackets-in-your-website)

**Comment by pdxryanpdx** [comment]
+1 i suck at coding so I cant do it but it could be very kool
